tests/src/Unit/PolicyVerificationTest.php

```diff
@@ -161,4 +175,25 @@ public function testAbstractPolicyCheckBaseRequirementError()
         $this->assertEquals('Could not proceed with policy verification due to requirement errors', $checkBaseMock->getResultMessage());
         $this->assertEquals(['Some data is missing'], $checkBaseMock->getRequirementErrors());
     }
+
+    /**
+     * Test a policy check with skip check.
+     *
+     * @return null
+     */
+    public function testExamplePolicyCheckSkipCheck()
+    {
+        /** @var \EdisonLabs\PolicyVerification\Test\ExamplePolicyCheckSkipCheck $checkBaseMock */
+        $checkMock = $this->getMockBuilder('EdisonLabs\PolicyVerification\Test\ExamplePolicyCheckSkipCheck')
+        ->setMethods([
+            'skipCheck',
+        ])
+        ->getMockForAbstractClass();
+
+        $checkMock->expects($this->any())
+        ->method('skipCheck')
+        ->willReturn(true);
+
+        $this->assertEquals(true, $checkMock->skipCheck());
+    }
```

Comment:
This is essentially testing that `willReturn()` works, which is not what we wan to do here.

What we want is to test that when `skipCheck` returns true, that `$checkMock->getChecks()` returns nothing. I'll commit shortly something to this effect.